web/templates/pages/tasks.html

```diff
@@ -0,0 +1,40 @@
+{% extends 'base.html' %}
+
+{% block title %}All CodeCatz Tasks{% endblock %}
+
+{% block content%}
+
+	<header class="row">
+		<div class="col-lg-6 col-sm-7">
+		      <h1>All CodeCatz Tasks</h1>
+		</div>
+	</header>
+
+	{% if task_list %}
+
+		<div class="row">
+
+			{% for task in task_list %}
+
+				<div class="col-md-3 col-xs-6">
+					<h3><strong>Task title: </strong>{{ task.title }}</h3>
```

Comment:
have you tried leaving the "Task title:" out? it should display just the actual title, I think that would be clear enough :thought_balloon: